### ISSUE TITLE
Error Unrecognized attribute: mask_filter_value for operator Attention

### ISSUE THREAD

**Issue by hammad26** [triton-inference-server/server#5547]
Hi
I converted my model into Onnx and then onnxruntime transformer optimization step is also done. Model is successfully loading and logits values are being matched with the native model as well. I moved this model to Triton server but facing following error on model loading step:

Error Unrecognized attribute: mask_filter_value for operator Attention

Library information is as:
```
onnx: 1.13.1
onnxruntime: 1.14.1
torch: 1.13.1
onnxruntime-tools:1.7.0
onnxconverter-common: 1.13.0
```

I tried two versions of triton inference server. Both gave the same errors:
```
nvcr.io/nvidia/tritonserver:21.04-py3
nvcr.io/nvidia/tritonserver:23.02-py3
```

Because onnxruntime model values(logits) are getting matched with the native model, so I am assuming that nothing wrong happened in conversion and optimization step. Would appreciate a quick help on it.

**Comment by jbkyang-nvi** [comment]
Hi @hammad26  can you share your model/config.pbtxt with us?

**Comment by hammad26** [comment]
Hi @jbkyang-nvi 
Sure. This is my config.pbtxt

```
name: "my_model"
platform: "onnxruntime_onnx"
input [
 {
    name: "input_ids"
    data_type: TYPE_INT64
    dims: [-1,200]
  },
  {
    name: "attention_mask"
    data_type: TYPE_INT64
    dims: [-1,200]
  },
  {
    name: "token_type_ids"
    data_type: TYPE_INT64
    dims: [-1,200]
  }
]
output [
{
            name: "output__0"
            data_type: TYPE_FP32
            dims: [-1,-1,1024]
          },
        {
                name: "output__1"
                data_type: TYPE_FP32
                dims: [-1,200]
              },
        {
                name: "output__2"
                data_type: TYPE_FP32
                dims: [-1,200]
              },
        {
                name: "output__3"
                data_type: TYPE_FP32
                dims: [-1,200]
              },
        {
                name: "output__4"
                data_type: TYPE_FP32
                dims: [-1,200]
              },
        {
                name: "output__5"
                data_type: TYPE_FP32
                dims: [-1,200]
              },
        {
                name: "output__6"
                data_type: TYPE_FP32
                dims: [-1,200]
              },
        {
                name: "output__7"
                data_type: TYPE_FP32
                dims: [-1,200]
              },
        {
                name: "output__8"
                data_type: TYPE_FP32
                dims: [-1,200]
              },
        {
                name: "output__9"
                data_type: TYPE_FP32
                dims: [-1,200]
              },
        {
                name: "output__10"
                data_type: TYPE_FP32
                dims: [-1,200]
              },
        {
                name: "output__11"
                data_type: TYPE_FP32
                dims: [-1,200]
              },
        {
                name: "output__12"
                data_type: TYPE_FP32
                dims: [-1,200]
              },
        {
                name: "output__13"
                data_type: TYPE_FP32
                dims: [-1,200]
              },
        {
                name: "output__14"
                data_type: TYPE_FP32
                dims: [-1,200]
              },
        {
                name: "output__15"
                data_type: TYPE_FP32
                dims: [-1,200]
              },
        {
                name: "output__16"
                data_type: TYPE_FP32
                dims: [-1,200]
              },
        {
                name: "output__17"
                data_type: TYPE_FP32
                dims: [-1,200]
              },
        {
                name: "output__18"
                data_type: TYPE_FP32
                dims: [-1,200]
              },
        {
                name: "output__19"
                data_type: TYPE_FP32
                dims: [-1,200]
              },
        {
                name: "output__20"
                data_type: TYPE_FP32
                dims: [-1,200]
              },
        {
                name: "output__21"
                data_type: TYPE_FP32
                dims: [-1,200]
              },
        {
                name: "output__22"
                data_type: TYPE_FP32
                dims: [-1,200]
              },
        {
                name: "output__23"
                data_type: TYPE_FP32
                dims: [-1,200]
              },
        {
                name: "output__24"
                data_type: TYPE_FP32
                dims: [-1,200]
              },
        {
                name: "output__25"
                data_type: TYPE_FP32
                dims: [-1,200]
              },
        {
                name: "output__26"
                data_type: TYPE_FP32
                dims: [-1,200]
              }]
```
